examples/model_service_template/model_handler.py

```diff
@@ -73,6 +73,7 @@ def handle(self, data, context):
         :param context: model server context
         :return: list of outputs to be send back to client
         """
+        self.error = None  # reset earlier errors
```

Comment:
I am curious, why do we need to explicitly reset error state? If this state is shared between requests then resetting it like this might not be safe. If it is not shared and each request has its own state for error - it is not clear why this explicit set to None is necessary.

Reply:
It's shared between multiple requests.
Each worker loads a model and created Service object. error is a member variable of the service object, so it need to be reset on each request.
Each work can only handle one request at a time, so it's safe to reuse service object.

Reply:
Oh, ok, I forgot that each worker can handle only one request at a time, my bad. I approve this PR then. Re: Vamshi's comment on it, I think it would be nice to revamp the code and make it a bit simpler (even if it becomes less OOP-like).